cocos/2d/CCFontAtlasCache.cpp

```diff
@@ -240,9 +229,11 @@ bool FontAtlasCache::releaseFontAtlas(FontAtlas *atlas)
 
 void FontAtlasCache::reloadFontAtlasFNT(const std::string& fontFileName, const Vec2& imageOffset/* = Vec2::ZERO*/)
 {
-    std::string atlasName = generateFontName(fontFileName, 0, false);
+    char tmp[255];
```

Comment:
I suggest to reuse the function `generateFontName` and refactor it to have only one args.